src/lib.rs

```diff
@@ -67,11 +67,11 @@ macro_rules! cfg_if {
     // Collects all the negated cfgs in a list at the beginning and after the
     // semicolon is all the remaining items
     (@__items ($($not:meta,)*) ; ) => {};
-    (@__items ($($not:meta,)*) ; ( ($($m:meta),*) ($($it:item)*) ), $($rest:tt)*) => {
-        // Emit all items within one block, applying an approprate #[cfg]. The
+    (@__items ($($not:meta,)*) ; ( ($($m:meta),*) ($($tokens:tt)*) ), $($rest:tt)*) => {
+        // Emit all items within one block, applying an appropriate #[cfg]. The
         // #[cfg] will require all `$m` matchers specified and must also negate
         // all previous matchers.
-        $crate::cfg_if! { @__apply cfg(all($($m,)* not(any($($not),*)))), $($it)* }
+        $crate::cfg_if! { @__apply cfg(all($($m,)* not(any($($not),*)))), $($tokens)* }
```

Comment:
```suggestion
        #[cfg(all($($m,)* not(any($($not),*))]   $crate::cfg_if! { @__identity $($tokens)* }
```

This is an (untested) suggestion to remove one layer of indirection  (then the @__apply part can be removed)

Reply:
Yeah, in my own testing of how you could convert it to use `:block` I was able to remove the `@__apply` macro thing as well. If Alex thinks we need to reduce it right away I can fix it, otherwise I'd rather just let that particular inefficiency wait for another day.

Reply:
Yeah this seems reasonable to go ahead and throw in!

Reply:
You were short a few `)` but it seems to work out and pass the tests.